src/Proxy.h

```diff
@@ -149,6 +149,7 @@ namespace sdbus::internal {
             {
                 std::unique_lock lock(mutex_);
                 auto asyncCallSlots = std::move(calls_);
+                calls_ = {};
```

Comment:
Is your goal to reallocate the map? If not, it might be cheaper to do `calls_.clear()`.